src/text_navigation.py

```diff
@@ -5,155 +5,200 @@
 ctx = Context()
 mod = Module()
 
-# With this you can set the maximum number of rows that will be included in the search,
-# for the keywords "above" and "below" in <user direction>   
-max_line_search = 10
 
-mod.list("cursor_location", desc="words to indicate if the cursor should be moved before or after a given reference point")
-mod.list("direction", desc="words to indicate a direction, as in: left, right, above, below")
-mod.list("navigation_option", desc="words to indicate type of navigation, for instance moving or selecting")
-mod.list("search_option", desc="words to indicate type of search, for instance matching a word with or without underscores")
+text_navigation_max_line_search = mod.setting(
+    "text_navigation_max_line_search",
+    type=int,
+    default=10,
+    desc="With this you can set the maximum number of rows that will be included in the search for the keywords above and below in <user direction>",
+)
+
+mod.list(
+    "cursor_location",
+    desc="words to indicate if the cursor should be moved before or after a given reference point",
+)
+mod.list(
+    "direction", desc="words to indicate a direction, as in: left, right, above, below"
+)
```

Comment:
You're not still using this anymore right?

Reply:
good catch